src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIRemoteProjectService.java

```diff
@@ -0,0 +1,132 @@
+package ca.corefacility.bioinformatics.irida.ria.web.services;
+
+import java.security.Principal;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.MessageSource;
+import org.springframework.stereotype.Component;
+
+import ca.corefacility.bioinformatics.irida.model.RemoteAPI;
+import ca.corefacility.bioinformatics.irida.model.project.Project;
+import ca.corefacility.bioinformatics.irida.model.project.ProjectSyncFrequency;
+import ca.corefacility.bioinformatics.irida.model.remote.RemoteStatus;
+import ca.corefacility.bioinformatics.irida.model.user.User;
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.RemoteProjectSettings;
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.RemoteProjectSettingsUpdateRequest;
+import ca.corefacility.bioinformatics.irida.service.ProjectService;
+import ca.corefacility.bioinformatics.irida.service.remote.ProjectRemoteService;
+import ca.corefacility.bioinformatics.irida.service.user.UserService;
+
+/**
+ * A utility class for doing operations on remote project sync settings.
+ */
+
+@Component
+public class UIRemoteProjectService {
+	private MessageSource messageSource;
+	private ProjectService projectService;
+	private ProjectRemoteService projectRemoteService;
+	private UserService userService;
+
+	@Autowired
+	public UIRemoteProjectService(MessageSource messageSource, ProjectService projectService,
```

Comment:
Same deal with this class.  These 2 methods are only used in one place.  Why add an extra class?

Reply:
Same deal here, please see above response.